supabase/migrations/20240102173707_organization_rls_update.sql

```diff
@@ -0,0 +1,19 @@
+drop policy "Enable read access for all users" on "public"."organization";
+CREATE POLICY "Enable read access for all users" ON public.organization AS PERMISSIVE FOR
+SELECT TO PUBLIC USING (
```

Comment:
it would be nice to have comments here that explain in plain text what each policy is doing

Reply:
first one
1. "Allows users to access all orgs they are a member of"
2. ??? "Allows users access all orgs they are resellers of"